### ISSUE TITLE
Open window on double click in system tray

### ISSUE THREAD

**Issue by rinaldus** [stanfieldr/ghetto-skype#122]
Is possible to add ability to open application window when user does double click on icon in system tray?

**Comment by stanfieldr** [comment]
Are you by chance running Unity? They don't support the click event, but other desktops such as Gnome, Mate and Cinnamon do. Or do you mean like a preference for single click vs double click?

**Comment by rinaldus** [comment]
No, I use KDE at home computer and GNOME at work computer. When I single left click on Ghetto Skype in KDE, context menu appears (the same as right click). Is possible to open chat (application window) on single (or double) left click?
